src/iframes/Pinger/js/auth.js

```diff
@@ -1,31 +1,31 @@
-/**
- * Created by michbil on 05.03.17.
- */
+// /**
+//  * Created by michbil on 05.03.17.
+//  */
```

Comment:
What does this change do?

Reply:
It was checking the auth. I assumed if we have guest users we don't need it.